mapserver/osm.map.mako

```diff
@@ -88,7 +83,7 @@ LAYER
     DATA "way FROM (SELECT ${columns} FROM planet_osm_point) AS foo USING UNIQUE osm_id USING srid=21781"
     FILTER ('[${layer['column']}]' ${layer.get('operator', '=')} '${layer['value']}')
     LABELITEM "name"
-    EXTENT 473743 74095 839000 306400
+    #EXTENT 473743 74095 839000 306400
```

Comment:
EXTENT already exists above, but I was not sure which one was the best one. I will keep this values and remove duplicate.